src/components/Bar_Main.js

```diff
@@ -10,10 +10,37 @@ export default class Bar_Main extends Component {
   }
 
   render() {
-    let { name } = this.props.cardData
+    let { name, imageURL, phone, notes, website, facebook, instagram, twitter } 
+      = this.props.cardData;
+    let { webIcon, phoneIcon, instaIcon, facebookIcon, twitterIcon}
+      = this.props.icons;
+
     return (
       <main className="Bar-Main Main_Info">
-        <p>Bar {name}</p>
+        <article className="Content">
+          <h2>{ name }</h2>
+          <img className="bar-img" src={ imageURL } alt="image of bar" />
```

Comment:
I think using the destructured `name`property might be a good fit for all our alt text, to provide a little more detail. Perhaps `${name} logo` or `image of ${name}`